lib/ansible/modules/windows/win_eventlog_entry.ps1

```diff
@@ -0,0 +1,121 @@
+#!powershell
+
+# (c) 2017, Andrew Saraceni <andrew.saraceni@gmail.com>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+# WANT_JSON
```

Comment:
You can get right rid of `# WANT_JSON` and `# POWERSHELL_COMMON` and replace it with just `#Requires -Module Ansible.ModuleUtils.Legacy.psm1`. This has more info as well http://docs.ansible.com/ansible/latest/dev_guide/developing_modules_general_windows.html#windows-new-module-development.